test/datasource/npm/index.spec.js

```diff
@@ -365,10 +365,12 @@ describe('api/npm', () => {
     expect(res1).toEqual(res2);
   });
   it('should use global cache', async () => {
-    const dummyValue = 'abc123';
-    await global.renovateCache.set('datasource-npm', 'foobar', dummyValue, 10);
+    const dep = {
+      name: 'abc123',
+    };
+    await global.renovateCache.set('datasource-npm', 'foobar', dep, 10);
     const res = await npm.getPkgReleases({ lookupName: 'foobar' });
-    expect(res).toEqual(dummyValue);
+    expect(res).toEqual(dep);
```

Comment:
Is this change necessary as part of this PR?

Reply:
I don't now, is from @JamieMagee. I can remove that.

Reply:
removed

Reply:
After reverted test, i found another bug:

```
TypeError: Cannot create property 'tags' on string 'abc123'
      at Object.getPkgReleases (lib/datasource/npm/releases.js:14)
```

The cache object should be an object 😄

Reply:
I'll commit it to master then and merge in here

Reply:
ok, I've made a seperate commit 😄

Reply:
#3599